### DISCUSSION TITLE
Github Copilot

### DISCUSSION THREAD

**Discussion by sadge1337** [community/community#65283] [Copilot Conversations]
is there any way to get copilot for free?

**Comment by Vette1123** [comment]
i think you can subscribe to it ( 10$ / month ), or if you're a student, you can get access after github verify that you're a student

**Comment by nirav-gajera** [comment]
If you are a student, you get access to GitHub Copilot for free through the[ Student Developer Pack](https://education.github.com/pack).

**Comment by sadge1337** [comment]
cool thanks guys
